essentials/scalar-variables/defined-or-operator/index.md

```diff
@@ -4,7 +4,7 @@ title: The defined-or operator
 
 {% include menu.html %}
 
-The so-called _defined-or_ operator `//` helps to assign a value to a variable that is not yet set.
+The so-called _defined-or_ operator `//` can be used to get a "fallback" value if a variable is not yet set.
```

Comment:
It does not assign anything, when I first read it I thought it would set value to the variable. But that is `//=`, not `//`